src/app/static/src/js/components/admin/manageRoles.vue

```diff
@@ -38,10 +42,24 @@
                     .then(({data}) => {
                         this.typeaheadEmails = data.data
                     })
+            },
+            roleAdded: function (role) {
+                const data = {name: role};
+                api.post(`/user-groups/`, data)
```

Comment:
I think this should now be `/roles/`. Which raises the point - we don't have integration tests between the front and back end for any of this new work. For the report page we use selenium tests, in hint we avoided that by adding integration tests for each api endpoint. We should probably think about doing one or other for the admin page.

Reply:
Ah yes, hadn't merged with latest master. 

Which approach would you prefer? Selenium tests are already set up, but on the other hand, they're Selenium tests.. Other approach would just require app to be running locally, and run integration js tests from the components that make the api calls? 

Have added a ticket for whichever approach we choose: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1333

Reply:
Thanks. Maybe we ought to stick with the selenium tests since we already have all that infrastructure set up? They're so slow though :sob: Lets have a think.